src/runner/main.cpp

```diff
@@ -425,12 +425,8 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
     {
         if (restart_if_scheduled() == false)
         {
-            auto text = is_process_elevated() ? GET_RESOURCE_STRING(IDS_COULDNOT_RESTART_NONELEVATED) :
-                                                GET_RESOURCE_STRING(IDS_COULDNOT_RESTART_ELEVATED);
-            MessageBoxW(nullptr, text.c_str(), GET_RESOURCE_STRING(IDS_ERROR).c_str(), MB_OK | MB_ICONERROR | MB_SETFOREGROUND);
-
-            restart_same_elevation();
-            result = -1;
+            Logger::warn("Scheduled restart failed. Trying restart as admin as fallback...");
+            restart_elevated();

```

Comment:
Why are trying to restart elevated? Seems strange that it would work for the issue people are having. How about doing a debug build to verify this.

Reply:
I tested it locally. The issue can easily be reproduced if you just try to install PowerToys on non-admin user.

The issue is happening because how we're trying to run non-elevated from elevated admin owned process. So we try to run non-elevated, if it fails, run elevated as it was working before.